### ISSUE TITLE
Window won't save placement and size

### ISSUE THREAD

**Issue by JohnnyFireOne** [mangerlahn/Latest#353]
Well everything works like a charm besides the window won't save placement and size it resets it self after quit and reopen the app

**Comment by mangerlahn** [comment]
Why did you close this? Has this resolved for you?

**Comment by JohnnyFireOne** [comment]
No, I thought its on the works so I close it to clear it up (silly me)

**Comment by mangerlahn** [comment]
I actually can't reproduce this issue. The current version restores the window size and position just fine. Which macOS are you on?

**Comment by JohnnyFireOne** [comment]
@mangerlahn

At 14.5 i tried a total cleaned install too and i have the same issue... whatever i do it's the same.
I’m using a Mac Studio M2 Max

**Comment by mangerlahn** [comment]
This should be resolved with 9f749b4
